src/test/groovy/org/prebid/server/functional/tests/ImpRequestSpec.groovy

```diff
@@ -184,6 +187,41 @@ class ImpRequestSpec extends BaseSpec {
         assert !bidderRequest?.imp?.first?.ext?.prebid?.bidder
     }
 
+    def "PBS should always update specified bidder imp when imp.ext.prebid.imp contain such bidder"() {
+        given: "PBs with openx bidder"
+        def pbsService = pbsServiceFactory.getService(
+                ["adapters.openx.enabled" : "true",
+                 "adapters.openx.endpoint": "$networkServiceContainer.rootUri/auction".toString()])
+
+        and: "Default basic BidRequest"
+        def impPmp = Pmp.defaultPmp
+        def extPrebidImpPmp = Pmp.defaultPmp
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            imp.first.tap {
+                pmp = impPmp
+                ext.prebid.bidder.openx = Openx.defaultOpenx
+                ext.prebid.imp = [(OPENX): new Imp(pmp: extPrebidImpPmp)]
+            }
+        }
+
+        when: "Requesting PBS auction"
+        def response = pbsService.sendAuctionRequest(bidRequest)
+
+        then: "Bid response should not contain warning"
+        assert !response?.ext?.warnings
+
+        and: "Generic bidderRequest should contain pmp from original imp"
+        def bidderToBidderRequests = getRequests(response)
+        assert bidderToBidderRequests[GENERIC.value].first.imp.pmp == [impPmp]
+
+        and: "OpenX bidderRequest should contain pmp from original imp"
+        assert bidderToBidderRequests[OPENX.value].first.imp.pmp == [extPrebidImpPmp]
+
+        and: "PBS should remove imp.ext.prebid.bidder from bidderRequests"
+        def bidderRequests = bidder.getBidderRequests(bidRequest.id)
+        assert !bidderRequests?.imp?.ext?.prebid?.imp?.flatten()
```

Comment:
Also add please assert for openxX bidder request.
bidderRequests[OPENX]?.imp.pmp